### PR TITLE
Sharable cart referral link - create new page

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

- created basic app for offers
- created empty state

## QA

- go to https://ubuntu-com-11050.demos.haus/advantage/offers
- check that it renders correctly

I can't manage to make the nav "suppressed" like we discussed @anthonydillon 

## Issue / Card

Fixes canonical-web-and-design/commercial-squad#434

## Screenshots

image

### CODE REVIEW

File: navigation.yaml [hunk 1]
```diff
@@ -1257,6 +1257,9 @@ support:
 
     - title: Community support
       path: /support/community-support
+    
+    - title: Your Offers
```
Comment (anthonydillon) [inline navigation.yaml:1261]:
```suggestion
    - title: Your offers
```

Comment (anthonydillon) [reply navigation.yaml:1261]: You can hide this menu with hidden: true

File: static/js/src/advantage/offers/Offers.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Row, Col } from "@canonical/react-components";
+import OffersList from "./components/OffersList";
+
+const Offers = () => {
+  return (
+    <div className="p-strip--suru-topped">
+      <Row>
+        <Col size={12}>
+          <h1>Your Ubuntu Advantage Offers</h1>
```
Comment (anthonydillon) [inline static/js/src/advantage/offers/Offers.tsx:10]:
```suggestion
          <h1>Your Ubuntu Advantage offers</h1>
```

**Review by anthonydillon — COMMENTED** [review]
Some small comments

File: navigation.yaml (hunk 1 above)
Comment (ClementChaumel) [reply navigation.yaml:1261]: But we discussed it being "suppressed" or only visible when you're on the page. The hidden setting hides it at all times.

File: static/js/src/advantage/offers/Offers.tsx (hunk 2 above)
Comment (ClementChaumel) [reply static/js/src/advantage/offers/Offers.tsx:10]: I followed the case on the wireframe. I did find it a bit odd to capitalise "offers" @wgx ?

**Comment by wgx** [comment]
@ClementChaumel good catch.. it's debatable. In our case an "Offer" is a named thing, but yeah please go ahead and downcase it to avoid controversy. :)

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**